plugin/al/schemas/AL/usd/schemas/maya/CMakeLists.txt

```diff
@@ -139,7 +143,7 @@ install(TARGETS _AL_USDMayaSchemas
 
 # configure_file has a nice feature where it will copy the __init__ file over when it gets modified, unlike file(COPY ...)
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/__init__.py ${CMAKE_BINARY_DIR}/AL/usd/schemas/maya/__init__.py COPYONLY)
-install(CODE "execute_process(COMMAND ${PYTHON_EXECUTABLE} -m compileall ${CMAKE_BINARY_DIR}/AL/usd/schemas/maya/__init__.py )")
+install(CODE "execute_process(COMMAND \"${PYTHON_EXECUTABLE}\" -m compileall ${CMAKE_BINARY_DIR}/AL/usd/schemas/maya/__init__.py )")
```

Comment:
I would also suggest wrapping the full path of the __init__.py in quotes as it is possible that CMAKE_BINARY_DIR could have a space in it.

Reply:
Same for all the other execute_process commands below. All the paths that are part of the COMMAND should be wrapped in quotes.